manifests/g/Grammarly/Grammarly/1.2.58.1269/Grammarly.Grammarly.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.12 by Dumplings $debug=QUSU.CRLF.7-2-17.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Grammarly.Grammarly
+PackageVersion: 1.2.58.1269
+PackageLocale: en-US
+Publisher: Grammarly Inc.

```

```suggestion
Publisher: Grammarly
```

```
Name          : Grammarly for Windows
Publisher     : Grammarly
Version       : 1.2.58.1269
ProductCode   : Grammarly Desktop Integrations
Registry Path : Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall\Grammarly Desktop Integrations
Uninstall     : winget uninstall --id "Grammarly Desktop Integrations"
```
Doesn't really matter when the ProductCode is used i guess..

---

Previously the installer wrote nothing but an empty string to the Publisher ARP property, so a placeholder was used here. On 1.0.34.630 (released on 2023-06-14) they seemed to fill the value when they were changing the application icon.

It doesn't really matter ig as I was still receiving updates for Grammarly from WinGet these months.